packages/shared/styles/components/organisms/SfStoreLocator.scss

```diff
@@ -31,7 +31,7 @@
     position: relative;
     width: var(--store-locator-map-width, 100%);
     max-width: var(--store-locator-map-max-width, none);
-    min-height: var(--store-locator-map-min-height, 26.875rem);
+    min-height: var(--store-locator-map-min-height, 34.5rem);
```

Comment:
On mobile view map takes all viewport height, wdyt to give the wrapper smaller value ?  It would be better to see possible lists of locations also.  :)

Reply:
good idea, changed to `75vh` :)